wiki/Manual.md

```diff
@@ -729,32 +729,45 @@ Each currency is an associative array (aka dictionary) with the following keys:
 
 ```JavaScript
 {
-    'id':      'btcusd',  // string literal for referencing within an exchange
-    'symbol':  'BTC/USD', // uppercase string literal of a pair of currencies
-    'base':    'BTC',     // uppercase string, unified base currency code, 3 or more letters
-    'quote':   'USD',     // uppercase string, unified quote currency code, 3 or more letters
-    'baseId':  'btc',     // any string, exchange-specific base currency id
-    'quoteId': 'usd',     // any string, exchange-specific quote currency id
-    'active':   true,     // boolean, market status
-    'taker':    0.002,    // taker fee rate, 0.002 = 0.2%
-    'maker':    0.0016,   // maker fee rate, 0.0016 = 0.16%
-    'percentage': true,   // whether the taker and maker fee rate is a multiplier or a fixed flat amount
-    'tierBased': false,   // whether the fee depends on your trading tier (your trading volume)
-    'feeSide': 'get'      // string literal can be 'get', 'give', 'base', 'quote', 'other'
-    'precision': {        // number of decimal digits "after the dot"
-        'price': 8,       // integer or float for TICK_SIZE roundingMode, might be missing if not supplied by the exchange
-        'amount': 8,      // integer, might be missing if not supplied by the exchange
-        'cost': 8,        // integer, very few exchanges actually have it
+    'id':      'btcusd',      // string literal for referencing within an exchange
+    'symbol':  'BTC/USD',     // uppercase string literal of a pair of currencies
+    'base':    'BTC',         // uppercase string, unified base currency code, 3 or more letters
+    'quote':   'USD',         // uppercase string, unified quote currency code, 3 or more letters
+    'baseId':  'btc',         // any string, exchange-specific base currency id
+    'quoteId': 'usd',         // any string, exchange-specific quote currency id
+    'active':   true,         // boolean, market status
+    'taker':    0.002,        // taker fee rate, 0.002 = 0.2%
+    'maker':    0.0016,       // maker fee rate, 0.0016 = 0.16%
+    'type':    'spot'         // spot for spot, future for expiry futures, and swap for perpetual swaps
+    'linear':   true          // the contract is a linear contract (settled in quote currency)
+    'inverse':  false         // the contract is an inverse contract (settled in base currency)
+    'contractSize': '1'       // the size of one contract, only used if `contract` is true
```

Comment:
While it's good to document this - i do feel that this is a huge missalignment within ccxt - and a huge bummer to usability (as such, this PR will not fix #11123).

We have the same exact naming (`'contractSize'`) in different endpoints, basically requiring users to "know" which endpoint returns which format (this is something that i'm yet to find any documentation on. I can guess this for most endpoints - but the documentation does not include any hint at "endpoint X returns the object Y".


As such, i consider this as a bug in ccxt (same attribute returns different values for different endpoints), which is not just a documentation problem.